scripts/ci/sandboxed_web_e2e.py

```
@@ -102,14 +102,15 @@ def start_service(label: str, command: str, cwd: Path, env: dict[str, str], logs
     """Start a service command in its own process group."""
     log_path = logs_dir / f"{label}.log"
     log_file = log_path.open("w", encoding="utf-8")
-    process = subprocess.Popen(
+    process = subprocess.Popen(  # nosec B603
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)